app/models/manageiq/providers/ansible_tower/automation_manager/workflow_job.rb

```diff
@@ -97,7 +97,7 @@ def update_with_provider_object(raw_workflow_job)
     update_parameters(raw_workflow_job) if parameters.empty?
 
     raw_workflow_job.workflow_job_nodes.each do |node|
-      next if node.job_id.nil?
+      next if !node.respond_to?(:job_id) || node.job_id.nil?
```

Comment:
would the following read better and do the same thing?

`next if node&.job_id.nil?`

Reply:
Yes, I was perhaps over-cautiously trying to use exactly what the ansible_tower_client uses: https://github.com/ansible/ansible_tower_client_ruby/blob/master/lib/ansible_tower_client/base_models/workflow_job_node.rb#L13

Reply:
Also `&.` doesn't handle method_missing like `.try()` does, so it would have to be `node.try(:job_id)`

Reply:
gotcha - makes sense

Reply:
Added a spec to cover the method_missing case, feel more comfortable with `.try()` now